packages/components/src/button/style.scss

```diff
@@ -13,7 +13,6 @@
 	align-items: center;
 	box-sizing: border-box;
 	padding: 6px 12px;
-	overflow: hidden;
```

Comment:
this was removed based on this comment: https://github.com/WordPress/gutenberg/pull/19058#discussion_r395537977
cc @tellthemachines